util/encrypt.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package util
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	"github.com/marcopeereboom/sbox"
+)
+
+// EncryptionKey wraps a 32 byte encryption key and the time when it
+// was created.
+type EncryptionKey struct {
+	Key  [32]byte // Key used for encryption
```

Comment:
This needs json field names.

Reply:
oof shouldn't have missed that one.  Fixed.